ci/tests/test_01_index_ci/test_01_malformed_index/test_index/test_01/test1.yml

```diff
@@ -0,0 +1,22 @@
+Projects:
+  - id: 1
+    app-id: test1
+    job-id: project1
+    git-url: https://github.com/mohammedzee1000/container-pipeline-service-indexci-testprojects
+    git-path: projects/baseproject/project1
+    git-branch: master
+    target-file: Dockerfile
+    desired-tag: latest
+    notify-email: mohammed.zee1000@gmail.com
+    depends-on: null
+
+  - id: 1
```

Comment:
Yes, its if you check corresponding test case that uses this particular index, you should notice its checking for duplicate id